tests/common/test_gtsfm_data.py

```diff
@@ -0,0 +1,149 @@
+"""Unit tests for the GtsfmData class.
+
+Authors: Ayush Baid
+"""
+import copy
+import unittest
+import unittest.mock as mock
+
+import gtsam
+import numpy as np
+from gtsam import Cal3Bundler, PinholeCameraCal3Bundler, Pose3, SfmTrack
+
+import gtsfm.utils.graph as graph_utils
+import gtsfm.utils.io as io_utils
+from gtsfm.common.gtsfm_data import GtsfmData
+
+GTSAM_EXAMPLE_FILE = "dubrovnik-3-7-pre"
```

Comment:
it would help to explain what this scenario is here. is it 3 cameras? what does 7 represent?
these should go in a docstring or inline comment here